src/Tasks/Resources/Strings.resx

```diff
@@ -2673,6 +2673,12 @@
     <comment>{StrBegin="MSB4801: "}</comment>
   </data>
 
+  <!-- Some tasks are only supported on .NET Framework -->
+  <data name="TaskRequiresFrameworkFailure" xml:space="preserve">
+    <value>MSB4803: The task "{0}" is not supported on the .NET Core version of MSBuild. Please use the .NET Framework version of MSBuild.</value>
```

Comment:
@kathleendollard @livarcocc  Thoughts on wording here? I don't know of a good customer-facing way to distinguish between .NET Core MSBuild and full framework MSBuild.exe. Maybe we should include an aka.ms link to docs, where we can be wordy about it?

Reply:
cc @mairaw

Reply:
Yeah, that's tricky. I like the idea of linking to docs where this can be explained in more detail.

Reply:
How would I get that link?

Reply:
You can create one using http://aka.ms. You can put me as co-owner and ddfwlink as a security group.

Reply:
Yay. Okay, so now is there an existing doc I should link against?

Reply:
I don't think that explanation already exists. 
Maybe link to https://docs.microsoft.com/visualstudio/msbuild/msbuild for now.

I've filed a doc issue for the VS team: https://github.com/MicrosoftDocs/visualstudio-docs/issues/2016

Reply:
Thanks.